youtube_dl/extractor/qqmusic.py

```diff
@@ -119,6 +119,12 @@ def _real_extract(self, url):
             'upload_date': publish_time,
             'creator': singer,
             'description': lrc_content,
+            'subtitles': {
+                'zh-CN': [{
+                    'ext': 'lrc',
```

Comment:
It's not an [`.lrc` format](https://en.wikipedia.org/wiki/LRC_%28file_format%29) but just a plain text.